x-pack/plugins/security_solution/server/utils/build_validation/route_validation.ts

```diff
@@ -65,3 +67,14 @@ export const buildRouteValidationWithExcess =
         (validatedInput: A) => validationResult.ok(validatedInput)
       )
     );
+
+export const buildRouteValidationWithZod =
```

Comment:
Thoughts on if utils like these would be useful in the codegen/generic package at some point? Do you think we will generate route skeletons at some point (when creating new routes)?

Reply:
👍 I agree, organizing codegen-related utils into a separate package seems like a good idea. I'll address this in upcoming PRs.